WebAPIBookStore/WebAPIBookStore/Usecase/OrderUseCase.cs

```diff
@@ -0,0 +1,194 @@
+﻿using AutoMapper;
+using WebAPIBookStore.Consts;
+using WebAPIBookStore.Dto;
+using WebAPIBookStore.Enum;
+using WebAPIBookStore.Interfaces;
+using WebAPIBookStore.Models;
+
+namespace WebAPIBookStore.UseCase
+{
+    public class OrderUseCase
+    {
+        private readonly IUserRepository _userRepository;
+        private readonly IOrderRepository _orderRepository;
+        private readonly ICartItemRepository _cartItemRepository;
+        private readonly IMapper _mapper;
+
+        public OrderUseCase(
+            IOrderRepository orderRepository,
+            IUserRepository userRepository,
+            ICartItemRepository cartItemRepository,
+            IMapper mapper)
+        {
+            _userRepository = userRepository;
+            _orderRepository = orderRepository;
+            _cartItemRepository = cartItemRepository;
+            _mapper = mapper;
+        }
+
+        public Output Get()
+        {
+            var output = new Output();
+            var orders = _orderRepository.GetOrders();
+            if (orders == null)
+            {
+                output.Success = false;
+                output.Error = StatusCodeAPI.NotFound;
+                output.Message = "Not found order";
+                return output;
+            }
+
+            output.Success = true;
+            output.Data = orders; 
+            return output;
+        }
+
+        public Output GetById(int id)
+        {
+            var output = new Output();
+            var order = _orderRepository.GetOrder(id);
+            if (order == null)
+            {
+                output.Success = false;
```

Comment:
anh thấy dạng lỗi như này mình đang dùng khá nhiều, sao em ko define 1 hàm về not found, việc gán chỉ gọi hàm đó thôi là nó tự gán cho nó gọn code. tương tự các chỗ khác